### ISSUE TITLE
Text fragment support

### ISSUE THREAD

**Issue by ClassicOldSong** [TheNeuronProject/ef.js#13]
Now we have mounting points and list mounting points, which can handle all ef-components at ease. But times when people want only to put a line of text into the mounting points, things gets really hard. Maybe we can have text fragments supported, which can also be helpful when jsx(#3) gets supported.

For Example:

```js
const {t} = ef

const component = new (t`
>div
  -nodeMountingPoint
  +listMountingPoint
`)()

component.nodeMountingPoint = 'Hello World!'
component.listMountingPoint.push('Hello ef.js!')
component.listMountingPoint.push('Hello text fragments!')

component.$mount({target: document.body})
```
could be rendered as:
```html
<div>Hello World!Hello ef.js!Hello text fragments!</div>
```
while with jsx support, we can have:
```jsx
const {t} = ef

const Div = t`
>div
  +children
`

const H1 = t`
>H1
  +children
`

const component = <Div>
  First line text
  <H1>Title text here</H1>
  Third line text
</Div>

component.$mount({target: document.body})
```

**Comment by ClassicOldSong** [comment]
With new version v0.9.0, `TextFragment` is now implemented, but in a slightly different approach:
```js
component.nodeMountingPoint = new TextFragment({text: 'Hello World!'})
component.listMountingPoint.push(new TextFragment({text: 'Hello ef.js!'}))
component.listMountingPoint.push(new TextFragment({text: 'Hello text fragments!'}))
```
TextFragment support for jsx is not yet fully supported.

**Comment by ClassicOldSong** [comment]
Fixed in v0.9.1
